content/en/blog/posts/2023-11-07-beta-api.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Introducing Shipwright Beta API"
+date: 2023-11-07T16:00:00-04:00
+draft: false
+author: "Enrique Encalada ([@qu1queee](https://github.com/qu1queee))"
+---
+
+About a year ago, we published a blog [post](https://shipwright.io/blog/2022/10/25/bringing-shipwright-to-beta-and-beyond/)
+in which we outline our vision and our values. Part of this vision was to advance
+our API to enhance its simplicity and consistency, and to signal a higher level of maturity.
+
+Today, as part of our release _v0.12.0_, we are introducing our _beta_ API. 
+The _beta_ API brings multiple changes as a result of accumulated experience operating
+the _alpha_ API and incorporating valuable user feedback.
+
+With the introduction of the _beta_ API, users can have confidence that our
+core components have been battle-tested, and using our different features is
+considered a safe practice.
+
+_We want to thank our community for their contributions and support in redefining this new API!_
+
+## Beta API
+
+The _beta_ API is available starting from the _v0.12.0_ release. The release is available 
+across our [cli](https://github.com/shipwright-io/cli), [operator](https://github.com/shipwright-io/operator) and [build](https://github.com/shipwright-io/build) repository.
+
+Within the _v0.12.0_ release, a conversion webhook has been introduced to ensure backward
+compatibility between the _v1alpha1_ and _v1beta1_. 
+
+We strongly encourage both current and future users to adopt the _beta_ API to benefit from
+its enhanced definition.
+
```

Comment:
Do we have any other notable new features? Deprecations?

Reply:
very few, like improvements in terms of security for the pods running on the `shipwright-build` namespace. I think is fine not adding them here, as this one focus more on the beta API only.